SearchHistory/tests.py

```diff
@@ -2,14 +2,14 @@
 from datetime import datetime
 import pytest
 
-PROFESSIONAL_ID = 101
-CLIENT_ID = 601
+PROFESSIONAL_ID = "professional1"
+CLIENT_ID = "client1"
 DATE = datetime(2023, 4, 17, 10, 0, 0)
 
 
 @pytest.fixture
-def searchHistory():
-    return SearchHistory(professional_id=PROFESSIONAL_ID, client_id=CLIENT_ID, date=DATE)
+def searchHistory(professional, client):
+    return SearchHistory(professional_id=professional, client_id=client, date=DATE)
```

Comment:
You already put it in the contest.py file...

Reply:
Done